tests/candev/Makefile

```diff
@@ -7,7 +7,10 @@ USEMODULE += isrpipe
 # define the CAN driver you want to use here
 CAN_DRIVER ?= native
 
-ifeq ($(CAN_DRIVER), PERIPH_CAN)
+ifeq ($(CAN_DRIVER), MCP2515)
+    USEMODULE += mcp2515
+
+else ifeq ($(CAN_DRIVER), PERIPH_CAN)
 # periph_can modules/variables go here
 
```

Comment:
I think this would be better handled with a `BOARD` based logic and to have it done in a `Makefile.board.dep`:

```
ifneq $(native, $(BOARD))
  USEMODULE += can_linux
else
  USEMODULE += mcp2515
endif
```

The test code should somehow exit early if there is no actual `mcp2515` connected.

Reply:
How would board based logic work here? There is no correlation between board and driver, except for native <-> Linux driver. 
Anyways, this has already been removed as was discussed in #14828.